scripts/reservationcharges.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# script specific variables
+working_dir=$(mktemp -d)
+IFS="|"
+data_source="reservation_charges"
+
+# Set date command based on OS
+platform=$(uname)
+if [[ $platform == "Darwin" ]]; then
+    date_command=$(which gdate)
+elif [[ $platform == "Linux" ]]; then
+    date_command=$(which date)
+fi
```

Comment:
could this be added to all the scripts not just this one please? you can probably extract it to one file and then source it

Reply:
added a functions.sh which is sourced by all scripts and moved this logic into it.